ruby/spec/find_cache_spec.rb

```diff
@@ -48,4 +48,20 @@
     expect(result).to be_success
     expect(result.resolved_key).to be_nil
   end
+
+  it 'respects key order' do
```

Comment:
The bucket now contains the following:
<img width="933" alt="Screen Shot 2020-12-14 at 09 18 42" src="https://user-images.githubusercontent.com/146013/102113065-6cfd1180-3ded-11eb-8ee1-c87c14c61916.png">

Reply:
Thanks I was trying to figure that out.